homeassistant/components/asuswrt/sensor.py

```diff
@@ -143,33 +141,24 @@ def __init__(
     ) -> None:
         """Initialize a AsusWrt sensor."""
         super().__init__(coordinator)
-        self._router = router
         self.entity_description = description
 
         self._attr_name = f"{DEFAULT_PREFIX} {description.name}"
         self._attr_unique_id = f"{DOMAIN} {self.name}"
-        self._attr_state_class = STATE_CLASS_MEASUREMENT
+        self._attr_device_info = router.device_info
+        self._attr_extra_state_attributes = {"hostname": router.host}
 
         if description.native_unit_of_measurement == DATA_GIGABYTES:
-            self._attr_last_reset = dt_util.utc_from_timestamp(0)
+            self._attr_state_class = STATE_CLASS_TOTAL_INCREASING
+        else:
+            self._attr_state_class = STATE_CLASS_MEASUREMENT
```

Comment:
This should be defined in AsusWrtSensorEntityDescription

Reply:
Fixed